aether/sdk/conf/settings.py

```diff
@@ -27,13 +27,22 @@ def get_required(name):
         raise RuntimeError(f'Missing {key} environment variable!')
 
 
+def env_bool(name, default=False):
```

Comment:
this is a breaking change :scream:

Reply:
I think the only case it _should_ is the negative, unless we're using other truthy values than 1, t, true.  The main consequence is that "false" no longer evaluates to True, which seems pretty confusing.